suite-native/module-send/src/sendOutputsFormSchema.ts

```diff
@@ -42,7 +43,16 @@ const isAmountHigherThanBalance = (amount: string, context?: SendFormFormContext
         return false;
     }
 
-    const { networkSymbol, networkFeeInfo, availableAccountBalance, isValueInSats } = context;
+    const {
+        networkSymbol,
+        networkFeeInfo,
+        availableAccountBalance,
+        isValueInSats,
+        precomposedFeeLevels,
+    } = context;
+
+    // transaction amount + normal fee has to be lower than account balance
```

Comment:
How is this comment relevant to the following line? 🤔 It's not obvious at all.

Reply:
better comment added in 5189ad143f